### DISCUSSION TITLE
Video OCR

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#66] [Ideas]
Extract code snippets from YouTube videos to use as context for your copilot.

**Comment by mason-at-pieces** [comment]
Super excited to see this in the product

**Comment by joshuadanpeterson** [comment]
Just saw this in the upcoming features tab. I can't wait for this. I watch a lot of YouTube tutorials, so this will be super helpful. You guys are doing amazing work!

**Comment by polyglotdev** [comment]
This looks awesome!! Can't wait to try it out!

**Comment by valentinobianco** [comment]
Looking forward to this too :)

**Comment by harshith-vaddiparthy** [comment]
Ability to add photos and snippets of videos could be great!

**Comment by REXREUS** [comment]
> Extract code snippets from YouTube videos to use as context for your copilot.

If it is successfully created it will certainly be very useful and beneficial. but there may be several problems such as consuming a lot of GPU and CPU power at the same time.

**Comment by miiraheart** [comment]
looking forward to this !
